app/build.gradle

```diff
@@ -105,4 +104,11 @@ dependencies {
     //Gson
     implementation gson.gson
 
+    // WorkManager
+    implementation workManager.workRuntime
+    implementation workManager.workRuntimeKtx
+
+    // WorkerInject
+    implementation injectWorker.workerInject
```

Comment:
Why not  usual WorkerFactory?
https://developer.android.com/reference/androidx/work/WorkerFactory

One more DI library with new annotations is not good. Especially if standard solution exists.
Let's use the classic way.

Reply:
Fixed